src/test/java/divaeva/hw/api/pet/GetPetTest.java

```diff
@@ -0,0 +1,46 @@
+package divaeva.hw.api.pet;
+
+import dto.CategoryDTO;
+import dto.ErrorResponse;
+import dto.PetDTO;
+import dto.TagDTO;
+import io.restassured.response.Response;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import rest.client.RestClient;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Random;
+import java.util.Set;
+
+public class GetPetTest extends PetTestBase {
+    private RestClient<PetDTO> restClient = new RestClient<>(BASE_SERVER_URL_PET);
+
+    @Test
+    public void getPetTest() {
+        int petId = 11;
+        PetDTO defaultPet = createDefaultPet(petId);
+        Response response = restClient.postEntity(defaultPet);
```

Comment:
Need status code verification  to be sure that entity is created successfully

Reply:
fixed